src/main/java/net/snowflake/ingest/streaming/internal/ChannelCache.java

```diff
@@ -82,14 +84,15 @@ void invalidateChannelIfSequencersMatch(
       String schemaName,
       String tableName,
       String channelName,
-      Long channelSequencer) {
+      Long channelSequencer,
+      String invalidationCause) {
```

Comment:
nit: in some methods it is invalidationCause and others it becomes errorMessage, would it be possible to reuse the same name so it is easier to follow?

Reply:
Agree that we should use invalidationCause or invalidationReason in the function signature, but sometimes a channel gets invalidated is because of an error, so it's ok to call it errorMessage for such cases when passing it into the function